### ISSUE TITLE
Another PR

### ISSUE THREAD

**Issue by serejandmyself** [cybercongress/cyber#22]
https://github.com/cybercongress/cyber/pull/21

There are a few unclear points that are left:
(brackets at each point, contain the mistake/proposed action)

1) mesh-network future proof (what was implied - possibly needs rephrasing?)
2) This is by far - not ideal, at least as 1/146, because we have 146 validators who verify the same computations producing the {\hyperref[knowledge-graph]{knowledge graph}}. (doesn't make sense, because of the word "because". Its either not ideal because of something or it is ideal because of something)
3) one content address can be voted on by a token - only once. (difficult to grasp - is that so?)
4) The period 'AdjustPricePeriod' sums up how much RC was spent during that period 'RecoveryPeriod' latest blocks. (broken logic)
5) This property prevents a situation where new and better content has a lower rank simply because it was submitted sometimes in the past. (doesn't make sense. possibly the other way around? The problem is "sometimes in the past". How can new content be submitted in the past? It is new, hence it is submitted now)
6) the search results can be easily delivered from any p2p network: e.g. .cyber can play videos (Does not have a wow effect. Needs expanding somehow)
7) So e-commerce can flourish fairly thanks to a provable conversion attribution (the message is unclear) 
8) We have decided to send {\hyperref[cyb]{CYB}} tokens to {\hyperref[thc]{THC}} staking manually (broken logic)
9) and accessible early {\hyperref[knowledge-graph]{knowledge graph}} gains in quality and size. (lost meaning)
10) e-Commerce has never been this easy for everyone. (unclear following / logic - need expanding)

**Comment by serejandmyself** [comment]
Closed
